src/atc/etl/loader.py

```diff
@@ -12,15 +14,29 @@ class Loader(EtlBase):
     and does not consume or change it.
     """
 
-    def __init__(self):
+    def __init__(
+        self, dataset_input_key: str = None, dataset_input_key_list: List[str] = None
```

Comment:
I dislike that it is possible to set both `dataset_input_key` and `dataset_input_key_list`. The behavior for a list of length 1 is the same as for a single key.

I recomment that the function should have the following signature 
```
def __init__(self, *dataset_input_keys: str):
```
this gives a list of string with length 0, 1 or more. 0 keys is consuming, 1 key results in a `save` and more keys is a `save_many`. This would make it unique, but still flexible.

Reply:
if you don't like the variadic style, then I propose to still only use the `dataset_input_key_list` which can optionally have a single key.

Reply:
I get you'r point. The reason why there are two parameters is to have full control of the save vs. save_many. When setting dataset_input_key i know save is called and when dataset_input_key_list i know save_many is called.
This is maybe not the best reason.

What about:
def __init__(self, dataset_input_keys: Union[str, List[str]] = None):

dataset_input_keys can both be string and list of strings, and save/save_many is called accordingly.
If dataset_input_keys is None, the original behaviour is used.

Reply:
@JeppeBlixen  - this PR has been here for some time. Have you implemented the above mentioned changes? ordo you need any assistance?

Reply:
Done